otwcode/otwarchive
# AO3-5046 AO3-5854 Modify OrphansController to improve handling of pseud orphaning.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5046
https://otwarchive.atlassian.net/browse/AO3-5854

## Purpose

- Rewrite `OrphansController#new` and `OrphansHelper#render_orphan_partial` so that the unused partials `orphans/_orphan_pseud` and `orphans/_orphan_user` are used.
- Rewrite the newly used partials so that they display a list of works to be orphaned, and have caution text echoing the existing text in `orphans/_orphan_work` (with a few slight changes).
- Rewrite `OrphansController#create` so that the `pseud_id` can be passed as an optional argument, to orphan only a single pseud (and make it so that only that pseud is removed, even if the user has another pseud on the same work).
- Add a new cucumber test for the changed behavior when orphaning a pseud.
- Add a collection of new RSpec tests for `OrphansController#new` and `OrphansController#create`.

## Testing Instructions

See JIRA for instructions for AO3-5046, and see the wiki for instructions on how to test AO3-5854.
